packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -539,7 +539,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       const sDate = d3Min(this._points, (point: IVerticalBarChartDataPoint) => point.x as Date)!;
       const lDate = d3Max(this._points, (point: IVerticalBarChartDataPoint) => point.x as Date)!;
       xBarScale = d3ScaleUtc()
-        .domain([sDate, lDate])
+        .domain(this._isRtl ? [lDate, sDate] : [sDate, lDate])
```

Comment:
the domain does not change. Change the range

Reply:
The fix was same in VSBC...

Reply:
10 to 1 in LTR vs 1 to 10 in RTL.... They're the same

Reply:
The same has been followed for NumericAxis

Reply:
Done